client/app/scripts/liveblog-edit/module.js

```diff
@@ -461,8 +461,14 @@ define([
         function canPublishAPost(blog) {
             return privileges.userHasPrivileges({'publish_post': 1});
         }
-        function isUserOwner(archive) {
-            if ($rootScope.currentUser._id === archive.original_creator || $rootScope.currentUser.user_type === 'administrator') {
+        function accessToSettings(archive) {
+            //if no role assigned, assume no restriction from the role POV
+            var rolePermission = true;
+            if ($rootScope.currentUser.role) {
+                rolePermission = rolePermission && privileges.userHasPrivileges({'blogs': 1});
+            }
+            if (($rootScope.currentUser._id === archive.original_creator && rolePermission) ||
```

Comment:
Editors also have the 'blogs' privilege but they only should have access to the settings of their own blogs while an administrator should be able to access the settings of all blogs